stable/nextcloud/Chart.yaml

```diff
@@ -0,0 +1,19 @@
+name: nextcloud
+version: 0.1.1
```

Comment:
Why is the chart version `0.1.1`? Shouldn't it be  `0.1.0` since it's a new chart?